src/middleware/HTTPMessageHandler.ts

```diff
@@ -27,11 +26,7 @@ export class HTTPMessageHandler implements Middleware {
 	 * @returns A promise that resolves to nothing
 	 */
 	public async execute(context: Context): Promise<void> {
-		try {
-			context.response = await fetch(context.request, context.options);
-			return;
-		} catch (error) {
-			throw error;
-		}
+		context.response = await fetch(context.request, context.options);
+		return;
```

Comment:
I don't believe the return is required here

Reply:
Done